petastorm/tests/test_codecs.py

```diff
@@ -34,6 +34,16 @@ def test_numpy_codec(self):
                                nullable=False)
         np.testing.assert_equal(codec.decode(field, codec.encode(field, expected)), expected)
 
+class NumpyArrayCodecsCompressionTest(unittest.TestCase):
+
+    def test_numpy_codec(self):
+        SHAPE = (10, 20, 30)
+        expected = np.random.rand(*SHAPE).astype(dtype=np.int32)
+        codec = NdarrayCodec(compression=True)
+        field = UnischemaField(name='test_name', numpy_dtype=np.int32, shape=SHAPE, codec=NdarrayCodec(),
```

Comment:
Should we use the same codec instead of creating a new instance? (`codec=NdarrayCodec()`)

Reply:
It is the same codec. test check compression option. Codec in UnischemaField is not used, exactly as in original test above.